src/recommender.py

```diff
@@ -24,8 +24,8 @@ def _get_playlist_json(self, tracks, location: QuackLocationType):
                 "id": track.id,
                 "name": track.name,
                 "artist": artists,
-                "image": "none"
+                "image": track.image
             }
             tracks_formatted.append(track_dict)
 
-        return service_response_playlist_json(tracks_formatted, location.name)
+        return service_response_playlist_json(tracks_formatted, location.name, offset)
```

Comment:
location i stedet for location.name, så der kan blive returneret en int i stedet for en string